SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -551,6 +554,25 @@ int64 USpatialActorChannel::ReplicateActor()
 		}
 	}
 
+	if (SpatialGDKSettings->bEnableUnrealLoadBalancer &&
+		NetDriver->LoadBalanceStrategy &&
+		// TODO: the 'bWroteSomethingImportant' check causes problems for actors that need to transition in groups (ex. Character, PlayerController, PlayerState),
```

Comment:
Is there a tracking ticket for this?

Reply:
Not that I am aware.

Reply:
Ideally we shouldn't commit commented out code. Can we just delete this and add it in later?

Reply:
Done